### DISCUSSION TITLE
First Time complexity and learning curve

### DISCUSSION THREAD

**Discussion by DanArlowski** [keptn/keptn#4754] [General]
On-Boarding for first time users is extremely difficult, 
the learning curve for keptn as I see it is quite high.
I suspect that is because the guide references are quite complicated

I think the documentation is not clear enough:
1. for creating the [prometheus](https://keptn.sh/docs/0.8.x/monitoring/prometheus/install/#configure-custom-prometheus-slis) SLI(/SLO? didnt understand if theres a difference between the configurations),
 for instance, the examples files there are not presented with names, which means that the commands in the end are quite general
 2. the CLI is quite complicated - flags for resource and resouceUri without explaining is not straightforward at all
 3. UI indications are quite complicated, when something is wrong when no SLI/SLO is presented, lighthouse should be errored IMHO.
  

I really wanted to evaluate and integrate Keptn, but currently the learning curve is too high to efficiently implement it.

I will reevaluate once it becomes more mature, im guessing at Version 1.0.

I really want it to become a discussion regarding how we can make the learning curve a little less steep 

and again, thanks for your contributions, it does not go unnoticed.

**Comment by jetzlstorfer** [comment]
Hi @DanArlowski 
thanks for opening this discussion.

I believe all of your points are valid and for us to better address them I'd like to ask how you got started with Keptn.
Did you first go through the [quickstart](https://keptn.sh/docs/quickstart/) guide or did you go to either the tutorials or the documentation?

We are working on providing an easy way to get started with Keptn but apparently, there is room for improvement :) 
It would help a lot if you can help us differentiate between bugs you encountered and bad user experience. With this, we can start addressing the pain points in the docs/tutorials/source. 

Also, where would you like to see more detailed information? Directly in the tutorials, the documentation? Or as indicators/messages in the CLI?

Thanks for your help!
